.github/workflows/lint.yml

```diff
@@ -0,0 +1,33 @@
+name: Lint
+
+on:
+  push:
+    paths:
+      - '**.py'
+      - '.github/ci/**'
+      - '.github/workflows/lint.yml'
+    branches:
+      - '**'
```

Comment:
OK to always run it?

Reply:
I dont think you need it on push on every branch tbh